bootcamp-milestone-2/src/app/components/blogPreview.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Blog } from "../blogData";
+import Link from "next/link";
+import style from "./blogPreview.module.css";
+
+export default function BlogPreview({ name, location, link }: Blog) {
+  return (
+    <div>
+      <Link href={link}>
+        <h3> {name} </h3>
+      </Link>
```

Comment:
I would consider adding styling on the links so that users know the link is clickable. 
I had to look into the code to figure out how to access your individual blogs.